cluster/libvirt-coreos/util.sh

```diff
@@ -220,6 +220,13 @@ function kube-up {
     rm $domain_xml
   done
 
+  export KUBECONFIG="$ROOT/.kubeconfig"
```

Comment:
I have amended the PR in order to reuse ${HOME}/.kube/.kubeconfig .
It new reuses the same config file as GCE but with a dedicated cluster name.